code/style.css

```diff
@@ -1,5 +1,248 @@
+margin-bottom: 16px;
+}
 
-body {
-  background-color: peachpuff;
+.sub {
+  font-family: impact, 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 24px;
+  font-weight: 500;
 }
+
+p {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-decoration: none;
+  color: black;
+}
+
+.container {
+  width: 100%;
+  flex-direction: column; 
+  max-width: 2000px;
+  text-align: center;
+}
+
+footer {
+  text-align: center;
+  text-decoration: none;
+  background-color: gainsboro;
+  color: black;
+}
+
+/*** HEADER ***/
+
+header {
+  background: white;
+  height: 80px;
+  display: flex; 
+  align-items: center;
+  padding: 0 16px;}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.hamburger div {
+width: 30px;
+height: 5px;
+background: black;
+border-radius: 5px;
+}
+
+header img {
+  width: 100px;
+  height: auto;
+  margin: 5px;
+}
+nav { 
+display: none; 
+}
+
+nav a {
+  color: black;
+  text-decoration: none;
+  margin-left: 20px;
+  font-family: monospace;
+  font-size: 15px;
+  
+}
+
+/*** FIRST CONTAINER ***/
+
+.first-container .container {
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.first-container img {
+    filter: brightness(130%);  
+    width: 300px;  
+    height: 300px;
+    border-radius: 50%;
+    object-fit: cover;
+    object-position: right;
+    flex-direction: column;
+    align-self: center;
+    padding: 20px;
+}
+
+.first-container div {
+  text-align: center;
+}
+
+/*** ALL BOXES ***/
+
+.box-container {
+  background-color: white;
+  color: black;
+  text-align: center;
+  padding: 24px;
+  width: 100%;
+  max-width: 2000px;
+}
+
+.box-container img {
+  width: 220px;
+  padding: 10px 10px 10px 10px;
+  border-radius:10%;
+  height: 200px;
+  object-fit: cover;
+}
+
+.boxes {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.boxes a {
+  background: #fafafa;
+  border: 5px solid black;
+  border-radius: 20px;
+  width: 100%;
+  text-decoration: none;
+  text-align: center;
+  color: black;
+  box-shadow: 12px 12px gainsboro;
+  padding: 20px 20px;
+  margin-bottom: 30px;
+}
+
+.boxes a div {
+  border: 2px solid; 
+  padding: 10px;  
+  border-radius: 10px;
+}
+ .circles {
+   display: flex;
+   justify-content: space-between;
+   margin: 16px;
+   margin-bottom: -16px;
+ }
+
+ .circles span {
+   width: 8px;
+   height: 8px;
+   border-radius: 50%;
+   background:  #367db8;
+ }
+
+ .circles span:last-child{
+background-color: black;
+ }
+
+
+/*** HOVER EFFECTS ***/
+
+
+.boxes a:hover {
+  background-color: yellow;
+}
+
+.box-container:hover {
+  filter: grayscale (100%);
+}
+
+
+@keyframes scale {
+  50% {
+      transform: scale(1.1);
+  }
+  100% {
+      transform: scale(1);
+  }
+}
+
+.boxes a:hover {
+  animation-name: scale;
+  animation-duration: 0.5s;
+}
```

Comment:
Love these hover effects! PS. Include them where you already started styling the same element (on line 189) so you don't have to write it twice:
.boxes a:hover {
  background-color: yellow;
  animation-name: scale;
  animation-duration: 0.5s;
}